docs/sources/mimir/references/architecture/query-sharding/index.md

```diff
@@ -18,6 +18,8 @@ query-frontend to return the full query result.
 
 ## Query sharding at glance
 
+Query sharding is applied on the [`query`]({{< relref "../../http-api/index.md#instant-query" >}}) and [`query_range`]({{< relref "../../http-api/index.md#range-query" >}}) APIs only.
```

Comment:
Pedantic and maybe subjective. I feel this line is better to be added above `## Query sharding at glance`  header. If we put it here, I feel there is a disconnection between this paragraph and the next paragraph.

It should be something like this below. I feel the continuity is better.

# Grafana Mimir query sharding

Mimir includes the ability to run a single query across multiple machines. This is achieved by breaking the dataset into smaller pieces. These smaller pieces are called shards. Each shard then gets queried in a partial query, and those partial queries are distributed by the query-frontend to run on different queriers in parallel. The results of those partial queries are aggregated by the query-frontend to return the full query result.

Query sharding is applied on the [query]({{< relref "../../http-api/index.md#instant-query" >}}) and [query_range]({{< relref "../../http-api/index.md#range-query" >}}) APIs only.

## Query sharding at glance

Not all queries are shardable. While the full query is not shardable, the inner parts of a query could still be shardable.

...

Reply:
hmm, good idea, moving up